ci/build/build-pipeline.yaml

```diff
@@ -859,10 +859,7 @@ spec:
         config:
           platform: linux
           params:
-            PROXY_NODE_URL: "https://proxy-node.test.eidas.signin.service.gov.uk"
-            STUB_CONNECTOR_URL: "https://stub-connector.test.eidas.signin.service.gov.uk"
-            STUB_IDP_USER: "stub-idp-demo-one"
```

Comment:
What's happened to `STUB_IDP_USER`? Was it redundant?

Reply:
The ruby script sets it to `stub-idp-demo-one` if not provided, so thought it wasn't necessary here

Reply:
👍